### PR TITLE
global.h was NEVER used. 

### PR DESCRIPTION
**Pull request by KjellKod**
global.h was NEVER used.  Instead it was realy using Research/source/i…nclude/global.h

The latter was split into NMGlobal.h which is installed by NMTools,. and Research/source/include/global.h which is ONLY used by Research

### CODE REVIEW

File: src/BoomStick.h [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #pragma once
 #include <string>
 #include <map>
-#include "global.h"
-#include "boost/uuid/uuid.hpp"
-#include "boost/uuid/uuid_generators.hpp"
+#include <NMGlobal.h>
```
Comment (john-gress) [inline src/BoomStick.h]: We should break the link between our builds (NMTools or Research) and QueueNado. We should just replicate the bit of global.h file needed by QueueNado in a QueueNadoGlobal.h file.

**Comment by john-gress** [comment]
Exclude the new QueueNadoGlobals.h file from the rpm package so it is not installed in /usr/local/probe/include

File: packaging/QueueNado.spec [hunk 2]
```diff
@@ -35,7 +35,7 @@ cp *.so $RPM_BUILD_ROOT/usr/local/probe/lib
 rm $RPM_BUILD_ROOT/usr/local/probe/lib/libgtest_170_lib.so
 mkdir -p $RPM_BUILD_ROOT/usr/local/probe/include
 cp src/*.h $RPM_BUILD_ROOT/usr/local/probe/include
-
+rm -f $RPM_BUILD_ROOT/usr/local/probe/include/src/QueueNadoMacros.h
```
Comment (KjellKod) [inline packaging/QueueNado.spec]: this doesn't work. Need to figure out why

File: src/QueueNadoMacros.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+#ifndef LR_FINAL
+#ifdef LR_DEBUG
+#define LR_FINAL 
+#else
+#define LR_FINAL final
+#endif
+#endif
+
+#ifdef LR_DEBUG
```
Comment (weberr13) [inline src/QueueNadoMacros.h]: how about QN instead of LR?

Comment (KjellKod) [inline src/QueueNadoMacros.h]: makes sense.

**Comment by KjellKod** [comment]
Took care of the comments.   Does it look OK guys?

**Comment by vrecan** [comment]
:+1:

**Pull request merged**